include/boost/config/compiler/borland.hpp

```diff
@@ -17,7 +17,7 @@
 #endif
 
 // last known compiler version:
-#if (__BORLANDC__ > 0x613)
+#if (__BORLANDC__ > 0x740)
```

Comment:
`0x740` is not a Borland compiler even

Reply:
Please look at the current Embacadero C++ compilers and you will realize that the latest release defines __BORLANDC__ and __CODEGEARC__ to 0x740. Really, sir, you need to do some invesigation before you make these assertions.

Reply:
This header is for classic Borland compiler and not for Embarcadero Clang-based not a Borland compiler, is not it?

Reply:
The header is used when \_\_CODEGEARC\_\_ is not defined, so it is for some older version of bcb. You are right that I could have left the last known compiler version at 0x613, but even that does not appear to be the actual last known version before \_\_CODEGEARC\_\_ was defined. So to play it safe I just allow the last known version to be whatever the current last version for Embarcadero C++ compilers now is.

Reply:
To play safe it should be reverted because now it means that __CODEGEARC__ compilers after 0x613 were tested and the `BOOST_NO_` macros filled

Reply:
I reverted it back to 0x613 in this PR.